src/dune_rules/workspace.ml

```diff
@@ -2,7 +2,9 @@ open Import
 open Dune_lang.Decoder
 module Repository = Dune_pkg.Pkg_workspace.Repository
 
-let default_repositories = [ Repository.overlay; Repository.upstream ]
+let default_repositories =
+  [ Repository.binary_dev_tools; Repository.overlay; Repository.upstream ]
```

Comment:
Given this is repo is very new, I think it would be nice to have this behind a feature flag and just enabled in the preview.

Personally I don't really want to pull binaries from the internet, so having an easy way to opt out of it (that is not putting `dune-workspace` files everywhere) seems important to me. Maybe an option in `dune-config` or so?

Reply:
Agreed, but do you mind if this comes in a later change?